src/test/utils/karate/businessconfig/postProcessGroups.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Process groups
+
+  Background:
+    # Get admin token
+    * def signIn = call read('../common/getToken.feature') { username: 'admin'}
+    * def authToken = signIn.authToken
+
+
+  Scenario: Post process groups file
+    Given url opfabUrl + '/businessconfig/processgroups'
+    And header Authorization = 'Bearer ' + authToken
+    And multipart file file = { read: 'resources/processgroups.json' }
+    When method post
+    And status 201
+
+
+  Scenario: Check that process groups have been updated
+    Given url opfabUrl + '/businessconfig/processgroups'
+    And header Authorization = 'Bearer ' + authToken
+    When method GET
+    Then status 200
+    And assert response.groups.length == 2
+    Then match response.groups[0].id == 'service1'
+    Then match response.groups[0].processes[0] == 'defaultProcess'
+    Then match response.groups[0].processes[1] == 'userCardExamples'
+    Then match response.groups[1].id == 'service2'
+    Then match response.groups[1].processes[0] == 'userCardExamples2'
+    Then match response.locale.en.service1 == 'Service number 1'
+    Then match response.locale.en.service2 == 'Service number 2'
+    Then match response.locale.fr.service1 == 'Service numéro 1'
+    Then match response.locale.fr.service2 == 'Service numéro 2'
```

Comment:
Just check status code is 200 . This way we can easly load new configuration with this feature just on modifying the processgoups.json

Reply:
Done